src/PendingCalls/TestCall.php

```diff
@@ -233,6 +233,40 @@ private function skipOn(string $osFamily, string $message): self
             : $this;
     }
 
+    /**
+     * Skips the current test unless the given test is running on Windows.
+     */
+    public function onlyOnWindows(): self
+    {
+        return $this->onlyOn('Windows', 'This test is skipped unless on [Windows].');
```

Comment:
Can you call `skipOnLinux` and `skipOnMac` instead?

Reply:
I think it's more fluent and easier to understand. 
If I have to alternative tests, one for a plaform and one for all the others, I can do onlyOnX on the first and skipOnX on the second. 
Moreover this will help in case other oses will be added. I'm using a bsd machine to launch  stress tests (using a patched stressless) and in that case i should skip 3 oses. If this get approved I'm planning to propose a PR also for skipOnBSD and onlyOnBSD.

Reply:
Sorry, I didn't understood what requested by Nuno. I'll revert the changes 🙃

Reply:
@nunomaduro This [commit](https://github.com/pestphp/pest/pull/1012/commits/f7705fe1c1c007aa91aac9333f5039f2ae6c4908) solve that.